### DISCUSSION TITLE
Internationalization

### DISCUSSION THREAD

**Discussion by ghost** [davmac314/dinit#210] [Ideas]
Please think carefully about adding this feature to dinit on behalf of a user who doesn't speak English, which some people call the "international language".

There are many countries that are not included in Europe, the United States and Australia. These users will prefer to use their native language over English, which has yet to be learned before the program can be used! Personally, I would also prefer to use the program in my native language than to help the Nazi collaborator Henry Ford, who had the slogan: "Make everybody speak English!"

Therefore, please give us the opportunity to use our native language when using the programs!

**Comment by davmac314** [comment]
It is planned - post 1.0 release, but it is planned. I am aware of how frustrating it must be to be forced to learn a new language just to use some software. :)

**Comment by ghost** [comment]
If the current version is 0.16, then roughly when will we see 1.0?

**Reply by davmac314** [reply]
"When it's ready". Sorry, can't really give a better answer than that. I work on it when I have time available, but it's hard to predict software completion milestones even in the best of circumstances.

I'm hoping to release 0.17 soon, that will be the beta release, maybe within the next month. If there are no major bugs 1.0 would be the next release after that. But as I said, it's hard to predict how long that will take to get to that stage.

**Comment by ghost** [comment]
Thank you very much! We'll be looking forward to it!

**Reply by davmac314** [reply]
Just to be clear, in case it wasn't: post 1.0 means "after" 1.0. I.e. the 1.0 release itself will probably not be internationalised, sorry.
